### PR TITLE
(NR-114641) Create opentelemetry-collector-prometheus.mdx

### PR DESCRIPTION
**Pull request by mangulonr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by akristen** [comment]
Hi @mangulonr ! I've moved this to our Draft column in our workflow. When this is ready to merge, reach out to our Docs help channel and we'll get it in one of our releases. :)

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      receivers: [prometheus]
+      processors: [batch, memory_limiter]
+      exporters: [otlp]
+```
+
+For further configuration options, please review:
+
+* [Opentelemetry Collector docs](https://opentelemetry.io/docs/collector/configuration/).
+* [Prometheus configuration docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) for additional configuration details.
+* [Scrape config doc](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config).
```
Comment (sigilioso) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx:93]:
```suggestion
* [Scrape config doc](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Update or create a new file called `config.yaml` from the example below.
+
+Replace the following keys in the file with your own values:
+
+* Prometheus scrape config:
+    * `PROMETHEUS_JOBNAME`
+    * `PROMETHEUS_HOSTNAME`  
+    * `PROMETHEUS_ENDPOINT_PORT`
+    * `PROMETHEUS_ENDPOINT_PATH`  (optional, default is /metrics)
```
[the remark this answers is not in the document]
Comment (mangulonr) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx:52]: I added to the example, now it is in both places.
I just left the optional keywork when enumerating the parameters.

**Review by nbaenam — APPROVED** [review]
Thank you for the update! I've added some minor modifications.

**Pull request merged**